cmd/loop/quote.go

```diff
@@ -59,6 +62,18 @@ func quoteIn(ctx *cli.Context) error {
 		return err
 	}
 
+	// For loop in, the fee estimation is handed to lnd which tries to
+	// construct a real transaction to sample realistic fees to pay to the
+	// HTLC. If the wallet doesn't have enough funds to create this TX, we
+	// don't want to fail the quote. But the user should still be informed
+	// why the fee shows as -1.
+	if quoteResp.MinerFee == int64(loop.MinerFeeEstimationFailed) {
+		_, _ = fmt.Fprintf(os.Stderr, "Warning: Miner fee estimation "+
```

Comment:
Do we generally use `stderr` for such messages in the cli?

Reply:
The idea behind this is that because the output is JSON, we would make it invalid if anyone is piping the output into a JSON parser. Then again, people should use the gRPC interface instead of parsing the command line output, so it's perhaps not worth the hassle.